v2/index.html

```diff
@@ -1618,20 +1618,18 @@ <h4>Grace Notes</h4>
             </p>
             <p>
                 For a single acciaccatura the lower-case character <code>g</code>
-                MUST be used. This character MUST be specified in the order given above;
-                that is, it MUST precede the note name, and MUST also precede all other
-                optional attributes of the note. Consecutive single acciaccatura
-                SHOULD NOT occur.
+                MUST be used. This character MUST precede the note name, and MUST
+                also precede all other attributes of the note. Consecutive single
+                acciaccatura MUST NOT occur.
             </p>
             <p>
                 For a single appogiatura note, the lower-case character <code>q</code>
-                MUST be used. This character MUST be specified in the order given above;
-                that is, it MUST precede the note name, and MUST also precede all other
-                optional attributes of the note.
+                MUST be used. This character MUST precede the note name, and MUST
+                also precede all other optional attributes of the note.
             </p>
             <p>
-                Multple appogiatura notes SHOULD be encoded using an <a href="#appogiatura-groups">
-                Appogiatura Group</a>. Consecutive single appogiatura SHOULD NOT occur.
+                Consecutive appogiatura notes MUST be encoded using an <a href="#appogiatura-groups">
+                Appogiatura Group</a>. Consecutive single appogiatura MUST NOT occur.
```

Comment:
```suggestion
                Appogiatura Group</a>.
```

These two sentences seem redundant?